packages/commerce-sdk-react/src/hooks/types.ts

```diff
@@ -226,3 +226,9 @@ export type TMutationVariables = {
     parameters?: {[key: string]: string | number | boolean | string[] | number[]}
     headers?: {[key: string]: string}
 } | void
+
+export type ServerContext = {
+    req: {
+        originalUrl?: string
+    }
+}
```

Comment:
please update the readme so link this change so people know when this was added 👍